src/main/kotlin/me/serce/solidity/lang/resolve/ref/SolImportPathReference.kt

```diff
@@ -13,13 +14,27 @@ class SolImportPathReference(element: SolImportPathElement) : SolReferenceBase<S
       return null
     }
     val path = importText.substring(1, importText.length - 1)
-    val file = element.containingFile.virtualFile.findFileByRelativePath("../$path")
+    val virtualFile = element.containingFile.virtualFile
+    val file = if (path.startsWith(".")) {
```

Comment:
It seems like relative `path` doesn't always start with a `.`, e.g. I've seen code like
```
import "oraclize/usingOraclize.sol";

contract PriceTicker is usingOraclize, Owned {
```

But I think it's possible to use `node_modules` search as a fallback.

Reply:
Although, Truffle doesn't work when you use something like 'import "oraclize/usingOraclize.sol"'
But, users can use some other framework or compile directly.

I can close the PR and use my implementation as fallback.
Then will resubmit it

Reply:
Although, Truffle doesn't work when you use something like 'import "oraclize/usingOraclize.sol"'
But, users can use some other framework or compile directly.

I can close the PR and use my implementation as fallback.
Then will resubmit it. OK?